docs/ref/databases.txt

```diff
@@ -819,6 +819,24 @@ You should either supply both :setting:`HOST` and :setting:`PORT`, or leave
 both as empty strings. Django will use a different connect descriptor depending
 on that choice.
 
+Full DSN and Easy Connect
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If both :setting:`HOST` and :setting:`PORT` are empty you can use full DSN
```

Comment:
Maybe :thinking: :

_"Full DSN or Easy Connect string in a :setting:\`NAME\` field can be used if both :setting:\`HOST\` and :setting:\`PORT\` are empty."_